fiware-region-sanity-tests/resources/templates/test_report_template.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Sanity Check Result Report- Fiware Region Status</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+
+<style>
+body {
+    font-family: Calibri, "Trebuchet MS", sans-serif;
+}
+* {
+    word-break: break-all;
+}
+table, td, th, .dataid {
+    border: 1px solid #aaa;
+    border-collapse: collapse;
+    background: #fff;
+    padding: 1px 10px 1px 10px;
+}
+section {
+    background: rgba(0, 0, 0, 0.05);
+    margin: 2ex;
+    padding: 1ex;
+    border: 1px solid #999;
+    border-radius: 5px;
+}
+h1 {
+    font-size: 130%;
+}
+h2 {
+    font-size: 120%;
+}
+h3 {
+    font-size: 100%;
+}
+h4 {
+    font-size: 85%;
+}
+h1, h2, h3, h4, a[href] {
+    cursor: pointer;
+    color: #0074d9;
+    text-decoration: none;
+}
+h3 strong, a.failed {
+    color: #ff4136;
+}
+a.success {
+    color: #3d9970;
+}
+pre {
+    font-family: 'Consolas', 'Deja Vu Sans Mono',
+                 'Bitstream Vera Sans Mono', 'Monaco',
+                 'Courier New', monospace;
+}
+
+.test-details,
+.traceback {
+    display: none;
+}
+section:target .test-details {
+    display: block;
+}
+
+</style>
+</head>
+<body>
+    <div align="center">
+        <img src="https://forge.fiware.org/plugins/mediawiki/wiki/fiware/images/a/a5/FI-WARE_logo.png" width="20%" height="20%"/>
+        <h1>FIWARE REGIONS - SANITY TESTS</h1>
+    </div>
+    <h1>Overview</h1>
```

Comment:
I have been analyzing it and that is not trivial. The template is built by the HTML reporting lib, and this one hasn't got any var available to use as timestamp in the template.
The options are:
- HTML post-procesing after it generation, to put the timestamp in the correct place.
- Fork the Nose HTML Report project (*https://github.com/lysenkoivan/nose-html-reporting*) to telefonicaid and add this feature. I think that is the best solution. I am already implementing it in a local fork.